Denik CZ.js

```diff
@@ -0,0 +1,110 @@
+function detectWeb(doc, url) {
+		return "newspaperArticle";
+}
+
+function doWeb(doc, url) {
+	var arts = new Array();
+	scrape(doc);
+}
+
+function scrape(doc) {
```

Comment:
Given your translator structure, `scrape` is unnecessary and everything should be merged into `doWeb`. If you do want to support multiples (which we encourage), then this structure would be what you want